scripts/compile-test.js

```diff
@@ -0,0 +1,28 @@
+import { mkdirSync, readFileSync, writeFileSync } from 'fs';
+import path from 'path';
+import glob from 'tiny-glob/sync.js';
+import { compile } from '../src/compiler/index.js';
+
+const cwd = path.resolve(process.argv[2]);
```

Comment:
is this a leftover from testing?

Reply:
Could you then add a comment at the top explaining what this is useful for and how to use it?